src/microbe_stage/MicrobeHUD.cs

```diff
@@ -30,10 +30,22 @@ public class MicrobeHUD : CreatureStageHUDBase<MicrobeStage>
     [Export]
     public NodePath UnbindAllHotkeyPath = null!;
 
+    [Export]
+    public NodePath StrainBarPanelPath = null!;
+
+    [Export]
+    public NodePath StrainBarPath = null!;
+
+    [Export]
+    public NodePath StrainBarFadeAnimationPlayerPath = null!;
```

Comment:
I think this should maybe be moved to the base HUD as shouldn't big creatures be eventually be able to sprint as well? So it makes sense to put there rather than having to copy-paste all this stuff to the multicellular HUD.